src/AddressRegistry.Console.Extract/appsettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "ExtractProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "ExtractProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "SyndicationProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;",
+    "SyndicationProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;"
+  },
+
+  "DataDog": {
+    "Enabled": true,
```

Comment:
Set this to false by default, we enable this in config on deploy. Just to prevent local dev spamming datadog